### DISCUSSION TITLE
No vc-domain-verify return

### DISCUSSION THREAD

**Discussion by vader1359** [vercel/community#4606] [Help]
### Summary

When I try to add the domain export.nanoco.com.vn to my project domain, I don't see any vc-domain-verify code return but only cname.vercel-dns.com. like in the picture. After following the suggested steps, it said the dns is succesfully set but I cannot visit the domain.
I've tried with a different subdomain is export.nanohome.vn, it works properly.
Both domains use the same domain provider.
Why?

### Example

nanoco-export.vercel.app

### Steps to Reproduce

- Add domain export.nanoco.com.vn to the project
-  Follow the steps suggested
- Visit the page export.nanoco.com.vn and see it fails.

Setup like this works properly with export.nanohome.vn
![CleanShot 2023-10-25 at 13 02 40](https://github.com/vercel/community/assets/27997820/532841ca-74ec-45df-b76b-15161b8190fb)

**Comment by amyegan** [comment]
Hey there! Please follow the steps in our documentation to add a custom [domain](https://vercel.com/docs/projects/domains/add-a-domain#apex-domains) or [subdomain](https://vercel.com/docs/projects/domains/add-a-domain#subdomains) to your Vercel project. 

https://vercel.com/docs/projects/domains/add-a-domain#
